packages/presentation/lib/screen/home/home_bloc.dart

```diff
@@ -0,0 +1,117 @@
+import 'dart:math';
+
+import 'package:domain/models/check_number_params.dart';
```

Comment:
when downloading your project, this path is highlighted in red

Reply:
forgot to add this file in this pr. I'll add this in next push